build/package/Dockerfile.java-toolset

```diff
@@ -0,0 +1,17 @@
+FROM adoptopenjdk/openjdk11:ubi
+
+SHELL ["/bin/bash", "-o", "pipefail", "-c"]
+
+ENV GRADLE_VERSION=7.1.1
+
+RUN cd /opt && \
+    yum -y install unzip && \
```

Comment:
To reduce disk size, let's clean up a little. In other images we do `&& rm -rf /var/cache /var/log/dnf* /var/log/yum.*` after `yum install`.